collect_app/src/androidTest/java/org/odk/collect/android/instrumented/utilities/CustomSQLiteQueryExecutionTest.java

```diff
@@ -53,14 +52,15 @@
                     Manifest.permission.READ_EXTERNAL_STORAGE,
                     Manifest.permission.WRITE_EXTERNAL_STORAGE)
             )
-            .around(new ResetStateRule());
-
-    @Before
-    public void setUp() {
-        sqLiteDatabase = SQLiteDatabase.openOrCreateDatabase(DATABASE_PATH, null, null);
-        dropTable(TEST_TABLE_NAME);
-        dropTable(TEST_TABLE_NAME_2);
-    }
+            .around(new RunnableRule(() -> {
+                try {
+                    File dbPath = File.createTempFile("test", ".db");
```

Comment:
This is just testing our db helpers so it doesn't need to talk to our "real" db.